### PR TITLE
Bug 1351325 added host list for NO_PROXY

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1351325

I'll have to get an ack through the BZ or email to see that's all that's needed.

### CODE REVIEW

**Comment by bfallonf** [comment]
OK. I got two acks in the BZ. I'll move ahead with this.

@tpoitras @adellape @ahardin-rh Peer review please?

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/http_proxies.adoc#install-config-http-proxies[Working with HTTP Proxies]
|Added more information about the `*NO_PROXY*` variable.
%

File: install_config/http_proxies.adoc [hunk 1]
```diff
@@ -47,6 +47,37 @@ NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16 <1>
...
+* Master IP or hostname
+* Service CIDR
+* Cluster IP CIDR
+For the Docker service::
+* Registry service IP and hostname
++
+[NOTE]
+====
+Currently, using CIDR for IP addressing is not supported by `*NO_PROXY*`; you
+will need to add individual IP addresses for such values. For example, the
```
Comment (tpoitras) [inline install_config/http_proxies.adoc]: s/you will need to/you must

File: install_config/http_proxies.adoc [hunk 2]
```diff
@@ -47,6 +47,37 @@ NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16 <1>
...
+registry.
```
Comment (tpoitras) [inline install_config/http_proxies.adoc]: sentence fragment?

File: install_config/http_proxies.adoc [hunk 3]
```diff
@@ -47,6 +47,37 @@ NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16 <1>
...
+* Service IP CIDR
+* Cluster IP CIDR
+For node hosts::
+* Master IP or hostname
+* Service CIDR
+* Cluster IP CIDR
+For the Docker service::
+* Registry service IP and hostname
++
+[NOTE]
```
Comment (tpoitras) [inline install_config/http_proxies.adoc]: Opinion here, but having two note admonitions back to back always looks a bit wrong to my eyes (especially with the way they render in OpenShiftland).

I think the first NOTE could just be plain text, and leave the second one as an admonition.

File: install_config/http_proxies.adoc [hunk 4]
```diff
@@ -47,6 +47,37 @@ NO_PROXY=master.hostname.example.com,10.1.0.0/16,172.30.0.0/16 <1>
...
+====
+Currently, using CIDR for IP addressing is not supported by `*NO_PROXY*`; you
```
Comment (tpoitras) [inline install_config/http_proxies.adoc]: This seems to contradict what's listed above. You can't use CIDR for IP addressing of *what* exactly?

**Comment by tpoitras** [comment]
@bfallonf left some comments :zap:

**Comment by bfallonf** [comment]
@tpoitras Made some edits. Any more thoughts?

**Comment by tpoitras** [comment]
:vertical_traffic_light: Green light! LGTM! :traffic_light:

**Comment by bfallonf** [comment]
Thanks @tpoitras ! Merging.

**Comment by tpoitras** [comment]
:dancers: :skull: :light_rail:

**Pull request merged**